src/components/Checkbox.jsx

```diff
@@ -4,42 +4,35 @@ import classNames from 'classnames';
 import compose from 'recompose/compose';
 import withHandlers from 'recompose/withHandlers';
 import withState from 'recompose/withState';
-
```

Comment:
Commenting again on this because GitHub is hiding the previous comment: let's undo _all_ of the changes to this file since they are unrelated to the rest of the PR.